packages/perseus/src/widgets/group.jsx

```diff
@@ -1,21 +1,21 @@
-/* eslint-disable react/sort-comp */
 // @flow
 import {linterContextDefault} from "@khanacademy/perseus-linter";
 import classNames from "classnames";
 import * as React from "react";
-import _ from "underscore";
 
 import * as Changeable from "../mixins/changeable.jsx";
 import {ApiOptions} from "../perseus-api.jsx";
 import Renderer from "../renderer.jsx";
 
 import type {PerseusGroupWidgetOptions} from "../perseus-types.js";
 import type {
+    APIOptions,
+    ChangeFn,
     FocusPath,
     PerseusScore,
     WidgetExports,
     WidgetProps,
-} from "../types.js";
+} from "../types";
```

Comment:
**suggestion**: In Perseus, we still keep the extensions (see relative imports above).  I know Kevin's mentioned that when we move to TypeScript, that will go away, but I think for now it'd be good to stay consistent with the surrounding code. :)

Reply:
VS Code or ESLint must have changed it; I'll switch it back.